src/constans.js

```diff
@@ -33,3 +33,8 @@ export const POINT_TYPE = [
   'sightseeing',
   'restaurant'
 ];
+
+export const EVENT_LIST_CLASS_CONSTANTS = {
+  EVENT_ROLLUP_BUTTON: 'event__rollup-btn',
+  EVENT_EDIT: 'event--edit'
+};
```

Comment:
если это перечисление, то пиши как перечисление 
```suggestion
export const EventSelector= {
  ROLLUP: '.event__rollup-btn',
  EDIT: '.event--edit'
};
```